editors/code/package.json

```diff
@@ -268,6 +268,116 @@
+                    "light": "#A31515",
+                    "highContrast": "#CE9178"
+                }
+            },
+            {
+                "id": "ralsp.keyword",
+                "description": "Color for keywords",
+                "defaults": {
+                    "dark": "#569cd6",
+                    "light": "#0000FF",
+                    "highContrast": "#569CD6"
+                }
+            },
+            {
+                "id": "ralsp.keyword.control",
+                "description": "Color for control keywords",
+                "defaults": {
+                    "dark": "#C586C0",
+                    "light": "#AF00DB",
+                    "highContrast": "#C586C0"
+                }
+            },
+            {
+                "id": "ralsp.keyword.unsafe",
+                "description": "Color for unsafe",
+                "defaults": {
+                    "dark": "#FF3030",
+                    "light": "#FF1010",
+                    "highContrast": "#FF1010"
+                }
+            },
+            {
+                "id": "ralsp.function",
+                "description": "Color for functions",
+                "defaults": {
+                    "dark": "#DCDCAA",
+                    "light": "#795E26",
+                    "highContrast": "#DCDCAA"
+                }
+            },
+            {
+                "id": "ralsp.parameter",
+                "description": "Color for parameters",
+                "defaults": {
+                    "dark": "#9CDCFE",
+                    "light": "#001080",
+                    "highContrast": "#9CDCFE"
+                }
+            },
+            {
+                "id": "ralsp.builtin",
+                "description": "Color for builtins",
+                "defaults": {
+                    "dark": "#DD6718",
+                    "light": "#DD6718",
+                    "highContrast": "#DD6718"
+                }
+            },
+            {
+                "id": "ralsp.text",
+                "description": "Color for text",
+                "defaults": {
+                    "dark": "#D4D4D4",
+                    "light": "#000000",
+                    "highContrast": "#FFFFFF"
+                }
+            },
+            {
+                "id": "ralsp.attribute",
+                "description": "Color for attributes",
+                "defaults": {
+                    "dark": "#9FE9BF",
+                    "light": "#1F4B1F",
+                    "highContrast": "#108010"
+                }
+            },
+            {
+                "id": "ralsp.literal",
+                "description": "Color for literals",
+                "defaults": {
+                    "dark": "#BECEA8",
+                    "light": "#09885A",
+                    "highContrast": "#B5CEA8"
+                }
+            },
+            {
+                "id": "ralsp.macro",
+                "description": "Color for DFAF8F",
```

Comment:
Wouldn't be it `Color for macros`?

Reply:
Thank you, fixed in https://github.com/rust-analyzer/rust-analyzer/pull/1305.